assets/training/finetune_acft_hf_nlp/components_v2/pipeline_components/chat_completion/spec.yaml

```diff
@@ -54,6 +54,12 @@ inputs:
 
   # Finetuning parameters
   # Training parameters
+  max_seq_length:
+    type: integer
+    optional: true
+    default: -1
```

Comment:
Can you put the default as what was there before? 8192 for safe side?

Reply:
yes, updated